docs/troubleshoot/security.md

```diff
@@ -2,6 +2,78 @@
 
 Refer to vulerability management and release documentation [here](https://github.com/ratify-project/ratify/blob/dev/SECURITY.md).
 
+## Signature Validation
+
+Ratify signs all dev images and dev helm OCI artifacts with Notary Project and Sigstore Cosign signatures.
+
+### Verifying Notary Project Signature
+
+Please install `notation` from [here](https://notaryproject.dev/docs/user-guides/installation/cli/)
+
+The public certificate for verification can be found at `ratify.dev/.well-known/ratify-verification.crt`
+
+> The latest certificate for verification can always be found at `ratify.dev/.well-known/ratify-verification.crt`. Refer to [Certificate Versioning](#certificate-versioning) guidance for details on verifying older artifacts.
+
+```shell
+curl -LO ratify.dev/.well-known/ratify-verification.crt
+notation cert add --type ca --store ratify-verify ./ratify-verification.crt
+cat <<EOF > ./trustpolicy.json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "ratify-images",
+            "registryScopes": [
+              "ghcr.io/ratify-project/ratify-dev",
+              "ghcr.io/ratify-project/ratify-base-dev",
+              "ghcr.io/ratify-project/ratify-crds-dev",
+            ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:ratify-verify" ],
+            "trustedIdentities": [
+                "x509.subject: CN=ratify.dev,O=ratify-project,L=Seattle,ST=WA,C=US"
+            ]
+        }
+    ]
+}
+EOF
+notation policy import ./trustpolicy.json
+notation verify ghcr.io/ratify-project/ratify-dev:latest
+notation verify ghcr.io/ratify-project/ratify-chart-dev/ratify:0-dev
+```
```

Comment:
Should we add the expected verification output to this section so users can make sure the signature is correctly verified?

Reply:
Added sample outputs